src/components/common/RichText/RichText.tsx

```diff
@@ -93,8 +89,8 @@ export function RichText({
     base: "w-full",
     variants: {
       editable: {
-        true: "px-3.5 py-3 [&_.tiptap]:outline-none",
-        false: "ring-0 rounded-none bg-transparent",
+        true: "px-3.5 py-3 focus:outline-none focus:ring-2 focus:ring-blue-500",
+        false: "ring-0 rounded-none bg-transparent focus:outline-none focus:ring-0",
```

Comment:
```suggestion
        false: "ring-0 rounded-none bg-transparent focus-within:outline-none",
```
This should be all we need to avoid the unwanted focus styles!